intake_stac/catalog.py

```diff
@@ -354,7 +363,7 @@ def __init__(self, key, asset):
             name=key,
             description=asset.title,
             driver=driver,
-            direct_access=True,
+            direct_access='allow',
```

Comment:
In hindsight, this is far from obvious! It won't matter to anyone not running an Intake server.